source/client1.c

```diff
@@ -0,0 +1,229 @@
+// Client side implementation of UDP client-server model
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <netinet/in.h>
+#include <errno.h>
+#include <netdb.h>
+#include <time.h>
+
+
+ 
+#define PORT     8080
+#define MAXLINE 1024
+#define MIN_COUNT 1
+#define MAX_NODES 5
+
+
+struct node
```

Comment:
Add tab spaces at beginning of each variable inside the structure .

Reply:
This change has been made